repl/scala-2.10/src/main/scala/org/apache/spark/repl/SparkILoop.scala

```diff
@@ -1064,15 +1064,15 @@ class SparkILoop(
   private def main(settings: Settings): Unit = process(settings)
 }
 
-object SparkILoop {
+object SparkILoop extends Logging {
   implicit def loopToInterpreter(repl: SparkILoop): SparkIMain = repl.intp
   private def echo(msg: String) = Console println msg
 
   def getAddedJars: Array[String] = {
     val envJars = sys.env.get("ADD_JARS")
-    val propJars = sys.props.get("spark.jars").flatMap { p =>
-      if (p == "") None else Some(p)
-    }
+    if (envJars.isDefined)
+      logWarning("ADD_JARS environment variable is deprecated, use --jar spark submit argument instead")
```

Comment:
our style guide actually requires us to put `{ } around the content of an `if`. I'll do this myself when I merge don't worry.